### PR TITLE
Fix auto picking streaming

### PR DESCRIPTION
**Pull request by danielmarchan3**
The initialStep did not return an StepId which was messing with all the prerequisites of the steps that were insert during the streaming.

### CODE REVIEW

File: sphire/protocols/protocol_cryolo_picking.py [hunk 1]
```diff
@@ -73,15 +73,19 @@ def _defineParams(self, form):
...
+        stepId = self._insertFunctionStep(self.createConfigStep)
+        return stepId
 
     # --------------------------- STEPS functions -----------------------------
     def _pickMicrograph(self, micrograph, *args):
         """This function picks from a given micrograph"""
         self._pickMicrographList([micrograph], args)
 
     def _pickMicrographList(self, micList, *args):
+        print("------------MIC LIIIIIIIIIIIST SPHIRE PROTOCOLO------")
```
Comment (pconesa) [inline sphire/protocols/protocol_cryolo_picking.py:85]: We don't want this! ;-)

File: sphire/protocols/protocol_cryolo_picking.py [hunk 2]
```diff
@@ -73,15 +73,19 @@ def _defineParams(self, form):
...
+        print(micList)
         if not micList:  # maybe in continue cases, need to properly check
+            print("no llega a picar nunca")
```
Comment (pconesa) [inline sphire/protocols/protocol_cryolo_picking.py:88]: NOr this!

**Review by pconesa — CHANGES_REQUESTED** [review]
hehe, you left some prints.

**Review by pconesa — APPROVED** [review]

File: sphire/protocols/protocol_cryolo_picking.py (hunk 1 above)
Comment (danielmarchan3) [reply sphire/protocols/protocol_cryolo_picking.py]: Sorry :)

**Pull request merged**